syntax = 'proto3';

package shop;

service AuthenticationService {
  rpc Auth(AuthRequest) returns (AuthResponse) {}
}

message AuthRequest {
  string message = 1;
}

message AuthResponse {
  string message = 1;
}
